setup.md

```diff
@@ -98,6 +98,16 @@ pip install tensorflow>=2.5 seaborn scikit-learn pandas
 [pip](https://pip.pypa.io/en/stable/) is the package management system for Python software packages.
 It is integrated into your local Python installation and runs regardless of your operating system too.
 
+### Troubleshooting for Macs with Apple M1 chip
```

Comment:
```suggestion
### Troubleshooting for newer Macs
```

Reply:
[Apple](https://support.apple.com/en-us/HT211814) call them `Macs with Apple silicon`. We could use same word here.

Reply:
I went for Cunliang's suggestion